generators/plopfile.ts

```diff
@@ -30,7 +30,7 @@ export default (plop: NodePlopAPI) => {
         },
         {
           type: 'add',
-          path: '../themes/theme-b2c-tailwind/src/{{atomicGroup}}/{{kebabCase name}}.css',
+          path: '../packages/styles/src/{{atomicGroup}}/{{kebabCase name}}.css',
```

Comment:
Cool! I thought about it yesteday ~ and now it's updated 💯

Reply:
:ant: mind reader :eye: